commands/backfill/backfill.update-screen.function.js

```diff
@@ -0,0 +1,16 @@
+var Moment = require('moment')
+
+module.exports = (function (get, set, clear) {
+
+  	var exchangeService = get('lib.exchange-service')(get, set, clear)
+
+	return (trade_id, data) => { 
+
+		if (data !== undefined && typeof process.stdout.clearLine == 'function') {
+			process.stdout.clearLine();
+			process.stdout.write("Processed trades up to " + Moment(data.time).fromNow() + "." );
```

Comment:
do you think we can show here exactly how many trades we have processed? I think that might give a better sense of what the bot is exactly doing (and what is taking so long)

Reply:
Good call.. I will make that change soon. I was going to have it display the time that the last trades were received/processed, too.

Reply:
Done in 5261f02 !